pkgs/development/libraries/jcal/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, autoreconfHook
+, readline
+, which
```

Comment:
Sorry, my bad, which isn't being used and can be removed.

Reply:
No, it can't be compile without it. i test it yesterday and it Throw and error for me.(also readline)
See:
https://www.reddit.com/r/NixOS/comments/b1s53f/cannot_find_which_autogen_cannot_continue/